src/spec/expectations.cr

```diff
@@ -1,4 +1,58 @@
 module Spec
+  # A list of diff command candidates.
+  DIFF_COMMANDS = %w(gdiff diff)
+
+  # A diff command path to use in diff computation.
+  class_property diff_command : String? do
+    DIFF_COMMANDS.compact_map { |name| Process.find_executable(name) }.first?
```

Comment:
This is really inefficient (no short-circuiting and needless array). Should just be a loop with `return`. Or `.find`.

(woops, I accidentally deleted the previous copy of this comment)